Running slasher with Docker on Windows not running as documented

https://docs.prylabs.network/docs/prysm-usage/slasher/

**Under:** Running slasher with Docker on Windows

Was informed on discord I needed a --network="host" param added to get slasher started properly. 

So:

`docker run -it --network="host" -v c:/prysm:/data gcr.io/prysmaticlabs/prysm/slasher:latest`

instead of 

`docker run -it -v c:/prysm:/data gcr.io/prysmaticlabs/prysm/slasher:latest`

Otherwise I ran into a bunch of errors then the process died. Would hope someone else would test this to be sure but that fixed it for me. Here is the error output for without that param added while also running a beacon and multiple validators concurrently on the same win machine in docker:

docker run -it -v c:/prysm:/data gcr.io/prysmaticlabs/prysm/slasher:latest
[2020-05-08 04:35:59]  WARN flags: Using default mainnet config
[2020-05-08 04:35:59]  INFO node: Checking DB database-path=/home/.eth2
[2020-05-08 04:35:59]  INFO RPC-API listening on port address=:5000
[2020-05-08 04:35:59]  WARN You are using an insecure gRPC connection! Provide a certificate and key to connect securely[2020-05-08 04:35:59]  WARN beaconclient: You are using an insecure gRPC connection to beacon chain! Please provide a certificate and key to use a secure connection
[2020-05-08 04:35:59]  INFO beaconclient: Successfully started gRPC connection
[2020-05-08 04:35:59] ERROR beaconclient: Could not fetch sync status error=rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:4000: connect: connection refused"
[2020-05-08 04:35:59]  INFO beaconclient: Waiting for beacon node to be fully synced...
[2020-05-08 04:36:11] ERROR beaconclient: Could not fetch sync status error=rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:4000: connect: connection refused"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xd4b17a]

goroutine 79 [running]:
github.com/prysmaticlabs/prysm/slasher/beaconclient.(*Service).querySyncStatus(0xc00057a6c0, 0x11b4800, 0xc0005d0500)
        slasher/beaconclient/chain_data.go:67 +0x5ba
github.com/prysmaticlabs/prysm/slasher/beaconclient.(*Service).Start(0xc00057a6c0)
        slasher/beaconclient/service.go:180 +0x4df
created by github.com/prysmaticlabs/prysm/shared.(*ServiceRegistry).StartAll
        shared/service_registry.go:46 +0x23e

--- a/docs/prysm-usage/slasher.md
+++ b/docs/prysm-usage/slasher.md
@@ -44,16 +44,16 @@ This section provides instructions to run [slasher](https://github.com/prysmatic
 #### Running slasher with Docker on GNU\Linux, and macOS 
 
 ```text
-docker pull gcr.io/prysmaticlabs/prysm/slasher:latest
-docker run -it -v $HOME/.eth2:/data --name slasher \
-  gcr.io/prysmaticlabs/prysm/slasher:latest \
+docker pull gcr.io/prysmaticlabs/prysm/slasher:stable
+docker run -it --network="host" -v $HOME/.eth2:/data --name slasher \
+  gcr.io/prysmaticlabs/prysm/slasher:stable \
   --datadir=/data
 ```
 #### Running slasher with Docker on Windows
 
 ```text
-docker pull gcr.io/prysmaticlabs/prysm/slasher:latest
-docker run -it -v c:/prysm:/data gcr.io/prysmaticlabs/prysm/slasher:latest
+docker pull gcr.io/prysmaticlabs/prysm/slasher:stable
+docker run -it --network="host" -v c:/prysm:/data gcr.io/prysmaticlabs/prysm/slasher:stable
 ```
 
 #### Running slasher with Bazel on GNU\Linux, macOS, and ARM64

